config/everything.mk

```diff
@@ -174,7 +174,7 @@ $(OBJDIR)/obj/%.d : src/%.c
 	# Generating dependencies for C source $< to $@
 	#######################################################################
 	$(MKDIR) $(dir $@) && \
-$(CC) $(CPPFLAGS) $(CFLAGS) -M -MP $< -o $@.tmp && \
+$(CC) $(CPPFLAGS) $(CFLAGS) -M $< -o $@.tmp && \
```

Comment:
I've reverted this change. I removed the `-MP` flag at some point, but why exactly escapes me now, since this flag appears fine for macOS on both x64 and M1/M2 🙂